spec/features/play_game_spec.rb

```diff
@@ -0,0 +1,16 @@
+feature "player enters choice" do
+  scenario "choose rock, paper or scissors" do
+    visit('/')
+    fill_in :player_name, with: "Jo"
+    click_button "Submit"
+    choose "Rock"
+    click_button "Submit"
+    expect(page).to have_content("rock")
+  end
+end
+
+# feature "computer makes a choice" do
+#   scenario "computer generates choice" do
+#     expect(computer_choice).to have_content("rock") or have_content("paper") or have_content("scissors")
```

Comment:
Nice I can see where you are going with this test :) 
You're right the choice would be one of these options. 
Since our computer_choice is random, we can make use of a seed (srand) https://ruby-doc.org/core-2.4.0/Random.html to make the outcome deterministic. Then we can check this is the content on the page